gallery.html

```diff
@@ -0,0 +1,83 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Responsive Cake webpage</title>
+    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <!-- Add your markup here -->
+<header class="header">
+    <nav class="nav">
+        <div class="content">
+            <ul class="link">
+            <li class="active"><a href="index.html">ABOUT ME</a></li>
+       
+            <li><a href="order.html">Order</a></li> 
+            <li><a href="contact.html">CONTACT</a></li>
+            </ul>
+            <img src="/designs/headerpc.png">
+          </div>
+        </nav>
+</header>
+
+
+
+<div class="container">
+    <div class="item">
+      <h3>french cake</h3>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit atque hic eligendi</p>
+    </div>
+    <img class="icon" src="C:\Users\User\Documents\GitHub\HTML-CSS-Coursework-Week3\.github\image\istockphoto-841257072-612x612.jpg">
+      🍔
+    </span>
+    <div class="item">
+      <h3>Italian cake</h3>
+      <p>Quos sunt non labore ab dicta ea sequi error, sapiente asperiores quas.</p>
+    </div>
+    <img class="icon" src="C:\Users\User\Documents\GitHub\HTML-CSS-Coursework-Week3\.github\image\istockphoto-1336693912-612x612.jpg">
```

Comment:
careful using absolute paths when importing; the path you need to use is either relative to the file you'r ein (gallery.html), or absolute based on the root of your project directory (which your `.git` folder is)